R/utils.R

```diff
@@ -159,3 +159,5 @@ copy_env <- function(src, dest = new.env(parent = emptyenv()),
 
   dest
 }
+
+last <- function(x) tail(x, n = 1L)
```

Comment:
I used tail because of this behavior
```r
x <- list()
tail(x, n = 1)
#> list()
x[[length(x)]]
#> Error in x[[length(x)]] :
#>  attempt to select less than one element in integerOneIndex
```
But I can switch if you would prefer the error here.